docs/source/functional.rst

```diff
@@ -4,6 +4,8 @@
 torchaudio.functional
 =====================
 
+.. py:module:: torchaudio.functional
+
```

Comment:
What's the function of adding it for sphinx-gallery?

Reply:
This allows to refer to the module as a while from examples, like `:py:mod:`torchaudio.functional`.

Reply:
Got it. Looks nice.